jaxrs-client/README-source.adoc

```diff
@@ -33,6 +33,11 @@ include::../shared-doc/use-of-jboss-home-name.adoc[leveloffset=+1]
 include::../shared-doc/start-the-standalone-server.adoc[leveloffset=+2]
 // Build and Deploy the Quickstart
 include::../shared-doc/build-and-deploy-the-quickstart.adoc[leveloffset=+2]
+
```

Comment:
Please revert the READE-source.adoc changes, I am a still bit unsure of what will be the future direction wrt docs and the root page, most probably will add something "later" to ../shared-doc/build-and-deploy-the-quickstart.adoc